Libraries/dotNetRDF/Query/Builder/IQueryBuilder.cs

```diff
@@ -41,8 +41,13 @@ public interface IQueryBuilder
         /// <summary>
         /// Gets the builder associated with the root graph pattern.
         /// </summary>
+        [Obsolete("Please use Root property")]
         GraphPatternBuilder RootGraphPatternBuilder { get; }
         /// <summary>
+        /// Gets the builder associated with the root graph pattern.
+        /// </summary>
+        IGraphPatternBuilder Root { get; }
```

Comment:
Added a new alias and made the long property obsolete bc it simply seems overly verbose

Reply:
👍